### PR TITLE
Removed google analytics for dev

### PR DESCRIPTION
**Pull request by harish-sethuraman**
We might not need to track development information? 

We shall merge after we confirm that GA is working properly 
- https://github.com/reactjs/reactjs.org/pull/4224#discussion_r789882998

### CODE REVIEW

File: beta/src/pages/_app.tsx [hunk 1]
```diff
@@ -30,20 +30,24 @@ export default function MyApp({Component, pageProps}: AppProps) {
...
     <AppShell>
       <Component {...pageProps} />
-      <Script
-        strategy="lazyOnload"
-        src={`https://www.googletagmanager.com/gtag/js?id=${process.env.NEXT_PUBLIC_GA_TRACKING_ID}`}
-      />
-      <Script id="google-analytics" strategy="lazyOnload">
-        {`
+      {process.env.NODE_ENV === 'production' && (
+        <>
```
Comment (gaearon) [inline beta/src/pages/_app.tsx:34]: no need for a fragment here

Comment (harish-sethuraman) [reply beta/src/pages/_app.tsx:34]: There are two script tags? That is why I added fragment 🤔.

Comment (gaearon) [reply beta/src/pages/_app.tsx:34]: ahh I missed it. Ok then!

Comment (harish-sethuraman) [reply beta/src/pages/_app.tsx:34]: :)

**Pull request merged**